### ISSUE TITLE
Fix: FeatureBy Image Link Alt text Issue

### ISSUE THREAD

**Issue by Krish-Parekh** [usesend/useSend#296]
### What happened?

The current FeatureBy setup has an issue with the image alt text; if an image fails to load, it affects the website’s look and feel because of the alt text.

<img width="1278" height="475" alt="Image" src="https://github.com/user-attachments/assets/6dc0e3c4-8170-4418-94bf-af576150038a" />

Right now, I’ve trimmed the author name to just the initial character. Since these names are constant, we have a few options:

- Keep the current approach and move ahead.
- Add a new alt field in the data specifically for alt text.

<img width="1338" height="658" alt="Image" src="https://github.com/user-attachments/assets/df431d1a-f38f-43cb-912c-963615190b87" />

Let me know if this approach works for you, or if you want me to implement one of the other options before making a PR.

**Comment by KMKoushik** [comment]
@Krish-Parekh looks good, just make sure the letter is in the center of the box if possible
